spring-thrift-starter/src/main/java/ru/trylogic/spring/boot/thrift/beans/RequestIdLogger.java

```diff
@@ -16,7 +16,7 @@
   private static final String DEFAULT_MDC_KEY = "request_id";
   private static final String X_REQUEST_ID = "x-request-id";
 
-  public String getMDCKey() {
+  public static String getMDCKey() {
```

Comment:
Why we need to do it static? What if somebody want to change MDC key creation logic?